releases/v21.2.2.md

```diff
@@ -0,0 +1,212 @@
+- Fixed an incorrect `no data source matches prefix` error for queries that use a set-returning function on the right-hand side of a [`JOIN`](../v21.2/joins.html) unless `LATERAL` is explicitly specified. [#71445][#71445]
+- Fixed a bug where using [`CREATE TABLE AS ....`](../v21.2/create-table-as.html) with a source query that referred to a sequence would not work. [#71541][#71541]
+- Support was added for the `"{}"` format for array columns in [`COPY FROM STDIN WITH CSV`](../v21.2/copy-from.html). [#72693][#72693]
+- Fixed a bug which caused [`ALTER COLUMN TYPE`](../v21.2/alter-column.html) statements to incorrectly fail. [#71165][#71165]
+- Fixed potential descriptor corruption bug for tables with a column with a [`DEFAULT`](../v21.2/default-value.html) expression referencing a [`SEQUENCE`](../v21.2/create-sequence.html) and with an [`ON UPDATE` expression](../v21.2/create-table.html#on-update-expressions). [#72362][#72362]
+- Fixed a bug where [schema changes](../v21.2/online-schema-changes.html) running during node shutdown could sometimes fail permanently when they should not. [#72333][#72333]
+- Fixed a panic that could occur with invalid [GeoJSON](../v21.2/geojson.html) input using `ST_GeomFromGeoJSON/ST_GeogFromGeoJSON`. [#71309][#71309]
+- Fixed a bug where specifying [`IntervalStyle`](../v21.2/interval.html) or [`DateStyle`](../v21.2/date.html) on `options=-c...` in a [connection string](../v21.2/connection-parameters.html#connect-using-a-url) would fail, even if the `sql.defaults.datestyle.enabled` and `sql.defaults.intervalstyle.enabled` [cluster settings](../v21.2/cluster-settings.html) were set. [#72067][#72067]
+- Fixed a bug where [session variables](../v21.2/set-vars.html) passed in the [connection string](../v21.2/connection-parameters.html#connect-using-a-url) were case-sensitive. Now they are all correctly normalized to lower case. [#72067][#72067]
+- Fixed a bug where `atttypmod` in [`pg_catalog.pg_attributes`](../v21.2/pg-catalog.html) for [`DECIMAL`](../v21.2/decimal.html) types with precision but no width was incorrectly `-1`. [#72074][#72074]
+- Fixed a bug where the [`setval`](../v21.2/create-sequence.html#sequence-functions) function did not invalidate cached sequence values. [#71821][#71821]
+- Fixed a bug where when creating an object [default privileges](../v21.2/authorization.html) from users that were not the user creating the object would be added to the privileges of the object. This fix ensures only the relevant default privileges are applied. [#72410][#72410]
+- Fixed a bug where the browser tab was not updated when the SQL Activity tab was selected on the [Sessions](../v21.2/ui-sessions-page.html), [Statements](../v21.2/ui-statements-page.html), and [Transactions](../v21.2/ui-transactions-page.html) pages of the DB Console. [#72784][#72784]
```

Comment:
Does it really make sense to include this bug fix when the SQL Activity tab is a new feature for the release?